relay-gateway/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=relay-gateway
+PKG_VERSION:=HEAD
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/relay-gateway
```

Comment:
I think keeping it the way it is at least on the dev branch will help developers to make changes locally and verify.  With the github link, the changes will need to be pushed first in order to do the verification.

This is something we have done for other gateway applications like button-led-controller, motion-led-controller. On master branch, we have ensured to use the github repo url